src/main/java/org/minions/devfund/alejandro/movie/database/Actor.java

```diff
@@ -0,0 +1,62 @@
+package org.minions.devfund.alejandro.movie.database;
+
+import java.util.ArrayList;
+
+/**
+ * This class contain the actor information.
+ */
+public class Actor {
+
+    private String name;
+    private ArrayList<Movie> movies;
+
+    /**
+     * Constructor of the Actor class.
+     * @param name the name of the actor.
+     */
+    public Actor(final String name) {
+        this.name = name;
+        this.movies = new ArrayList<Movie>();
+    }
+
+    /**
+     * Constructor of the Actor class.
+     */
+    public Actor() {
+        this.name = " ";
+        this.movies = new ArrayList<Movie>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)